client/post-editor/media-modal/detail/detail-title.jsx

```diff
@@ -69,10 +69,6 @@ export default React.createClass( {
 				title: this.state.title
 			} );
 		}
-
-		if ( this.isMounted() ) {
-			this.resetTitle();
```

Comment:
Thanks @aduth for the clarification!

I think I found one regression @vvasilev-
1. Open the media modal
2. Select more than one image
3. Click Edit
4. Update the title
5. Click on the ">" control to go to the next image
6. The next image has the same title as the first.